recipe/meta.yaml

```diff
@@ -9,29 +9,29 @@ source:
   sha256: 7682dc8afb30297001674575ea00d1814d808d6a36af415a82bd481d37ba7b8e
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  number: 2
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  string: "win_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"   # [win]
+  string: "unix_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"   # [unix]
```

Comment:
Just so I can understand, was the `string` changed for convenience or is it serving some other purpose?

Reply:
You could get identical build strings even if you build on different platforms. I'd guess this is to guard against it.

Reply:
It's possible to avoid this issue though ( https://github.com/conda-forge/conda-forge.github.io/pull/1839#discussion_r1007959130 )